packages/lit-html/src/test/lit-html_test.ts

```diff
@@ -772,18 +771,12 @@ suite('lit-html', () => {
 
     test.skip('renders a Symbol to an attribute', () => {
       render(html`<div foo=${Symbol('A')}></div>`, container);
-      assert.include(
-        container.querySelector('div')!.getAttribute('foo'),
-        ''
-      );
+      assert.include(container.querySelector('div')!.getAttribute('foo'), '');
     });
 
     test.skip('renders a Symbol in an array to an attribute', () => {
       render(html`<div foo=${[Symbol('A')] as any}></div>`, container);
-      assert.include(
-        container.querySelector('div')!.getAttribute('foo')!,
-        ''
-      );
+      assert.include(container.querySelector('div')!.getAttribute('foo')!, '');
```

Comment:
Can we fix the lint issues below this in this file?

Reply:
done